src/main/resources/locales/en/plugins.json

```
@@ -0,0 +1,34 @@
+{
+  "scm-issuetracker-plugin": {
+    "title": "Issue Tracker",
+    "subtitle": "General Issue Tracker configuration",
```

Either all caps or not.
```suggestion
    "subtitle": "General issue tracker configuration",
```